src/commands/math/commands.js

```diff
@@ -485,6 +485,16 @@ CharCmds['/'] = P(Fraction, function(_, super_) {
         cursor[L] = leftward;
       }
     }
+    if (this.replacedFragment) {
+      if (this.replacedFragment.ends[L] instanceof Bracket &&
```

Comment:
Will this delete the `+c` if I'm here and I press `/`?
![Screen Shot 2020-09-14 at 20 31 18](https://user-images.githubusercontent.com/225809/93162337-716c3e00-f6c9-11ea-8559-31a1c03d892b.png)


```suggestion
      if (this.replacedFragment.ends[L] instanceof Bracket &&
        this.replacedFragment.ends[L] === this.replacedFragment.ends[R] &&
```

Reply:
Good catch.